### PR TITLE
Use log messages for metrics

### PR DESCRIPTION
**Pull request by andymutton**

### CODE REVIEW

**Comment by samuelli** [comment]
lgtm. Do we do any sort of linting/tests on analysis?

To use the ESLint that we run, just need to add it to to target src array in the `Gruntfile.js`.

**Comment by andymutton** [comment]
(I enabled lint and fixed all the lint errors)

File: analysis/ana_log.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      console.log(path);
+    }
+  }
+  static log() {
+    console.log(...arguments);
+  }
+}
+
+module.exports = {
+  Ana: Ana
```
Comment (dstockwell) [inline analysis/ana_log.js]: `module.exports = {Ana};`

Comment (dstockwell) [inline analysis/ana_log.js]: Although `module.exports = Ana;` is probably all you need, unless you expect to export something else later?

File: analysis/ana_log.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  static fail(counterPath) {
+    var path = counterPath + "/fail";
+    if (arguments.length > 1) {
+      console.log(path, ...Array.prototype.slice.call(arguments, 1));
+    } else {
+      console.log(path);
+    }
+  }
+  static log() {
+    console.log(...arguments);
```
Comment (dstockwell) [inline analysis/ana_log.js]: Do we need to escape this if the first arg ends with `/success` or `/fail`?

**Pull request merged**